docker-compose.yml

```diff
@@ -15,11 +15,12 @@ services:
       '"
 
   jh_image:
+    image: ${CLASS}:latest
     build:
       context: .
       target: ${TARGET}
       cache_from: 
-        - gcr.io/jupyterhub-docker-images/${CLASS}:latest
+        - pkg.dev/jupyterhub-docker-images/${CLASS}:latest
```

Comment:
Not sure if we need to add the region in. I think Isa's note had that comment

Reply:
should i remove that everywhere then?

Reply:
Not sure what you mean? What I meant is that we just need to use us-east1-docker.pkg.dev here. I think it's changed everywhere else

Reply:
oh that would make more sense. it sounded like you meant that the region marker wasn't necessary, and as such should be removed, and not that it might be necessary here for the caching to work properly. 

i tried a run with this without the region marker (which failed, unsurprisingly) but I'll add it back here and see if i can get a speed boost from caching properly.

Reply:
I didn't notice a time save from the cacheing, but maybe itll be better next time. I'm going to merge this and start on the next one then!